examples/Fast_Diffusion.jl

```diff
@@ -0,0 +1,46 @@
+# # Fast-Diffusion Problem
```

Comment:
this needs to be added to the tests. We really need to remove the examples folder and instead just write tutorials in the docs too.